### DISCUSSION TITLE
detected coimminer

### DISCUSSION THREAD

**Discussion by ThePloimTid** [dahall/TaskScheduler#931] [Q&A] [answered]
Why, when I add a function to create a task to my code, my program is detected as coinminer?

**Comment by dahall** [comment, marked as answer]
Really no idea. The code is simply wrapper functions around the windows task scheduler API. Over the many years this has been in use, occasionally it has been detected, as have most public libraries, as a false positive for different scanners. Please work with the scanner to qualify it as safe.
